src/utils/components/Charts/utils/queries.ts

```diff
@@ -33,14 +33,10 @@ type GetUtilizationQueries = ({ duration, launcherPodName, nic, obj }: Utilizati
   [key in VMQueries]: string;
 };
 
-export const getUtilizationQueries: GetUtilizationQueries = ({
-  duration,
-  launcherPodName,
-  obj,
-}) => {
+export const getUtilizationQueries: GetUtilizationQueries = ({ duration, obj }) => {
   const { name, namespace } = obj?.metadata || {};
   return {
-    [VMQueries.CPU_REQUESTED]: `sum(kube_pod_resource_request{resource='cpu',pod='${launcherPodName}',namespace='${namespace}'}) BY (name, namespace)`,
+    [VMQueries.CPU_REQUESTED]: `kubevirt_vm_resource_requests{resource='cpu',unit='cores',name='${name}',namespace='${namespace}'}`,
```

Comment:
this change seems not related to the link

Reply:
No ahahah i have to remove it